### PR TITLE
Remove hardcoded data from Hearing model

### LINKED ISSUE #9595: Hearing model | Remove hardcoded data
We left 3 hardcoded methods 😧 

AC:
1) Update regional_office_name and regional_office_timezone to use the location generated by the AHL job
1) Update current_issue_count to count the number of request_issues

### PR DESCRIPTION
**Pull request by oscarAtNava**
Resolves #9595

### Description
Removed hard coded data from Hearing model as specified in issue ticket.

### Acceptance Criteria
- [ ] Code compiles correctly

### Testing Plan
1. Go to Daily Docket
2. Confirm Hearings for RO display correctly.

### CODE REVIEW

File: app/models/hearing.rb [hunk 1]
```diff
@@ -90,19 +90,17 @@ def worksheet_issues
     end
   end
 
-  #:nocov:
-  # This is all fake data that will be refactored in a future PR.
   def regional_office_name
-    RegionalOffice::CITIES[regional_office_key][:label] unless regional_office_key.nil?
```
Comment (sharonwarner) [inline app/models/hearing.rb]: For this, we can use `RegionalOffice::CITIES[hearing_day.regional_office][:label] unless hearing_day.regional_office.nil?`

File: app/models/hearing.rb [hunk 2]
```diff
@@ -90,19 +90,17 @@ def worksheet_issues
...
+    hearing_location&.city
   end
 
   def regional_office_timezone
-    RegionalOffice::CITIES[regional_office_key][:timezone] unless regional_office_key.nil?
+    RegionalOffice::CITIES[appeal.closest_regional_office][:timezone] unless appeal.closest_regional_office.nil?
```
Comment (sharonwarner) [inline app/models/hearing.rb]: Actually, now that I'm thinking about it, using hearing_day.regional_office is a little cleaner here too. It's possible that a hearing coordinator could override the closest_regional_office.

Comment (oscarAtNava) [reply app/models/hearing.rb]: ok

File: app/models/hearing.rb (hunk 1 above)
Comment (sharonwarner) [reply app/models/hearing.rb]: I'm sorry! I didn't realize we had an alias set up for regional_office_key to return hearing_day.regional_office! This logic is actually good the way it was set up initially :( :(

File: app/models/hearing.rb [hunk 3]
```diff
@@ -90,19 +90,17 @@ def worksheet_issues
...
-    RegionalOffice::CITIES[regional_office_key][:timezone] unless regional_office_key.nil?
+    RegionalOffice::CITIES[hearing_day.regional_office][:timezone] unless hearing_day.regional_office.nil?
     "America/New_York"
   end
 
   def current_issue_count
-    1
+    request_issues.size
   end
   #:nocov:
```
Comment (sharonwarner) [inline app/models/hearing.rb]: Can we remove this #:nocov: as well?

**Pull request merged**